src/main/java/com/cloudbees/hudson/plugins/folder/Folder.java

```diff
@@ -373,59 +370,6 @@ public TopLevelItem newInstance(ItemGroup parent, String name) {
             return new Folder(parent, name);
         }
 
-        /**
-         * Auto-completion for the "copy from" field in the new job page.
-         */
-        public AutoCompletionCandidates doAutoCompleteCopyNewItemFrom(@AncestorInPath final Folder f,
```

Comment:
Was not actually being used, it turns out, and core is a better place to fix the details.